### ISSUE TITLE
Feedback: consider returning the callbacks in the setters

### ISSUE THREAD

**Issue by chmp** [gereleth/jupyter-bbox-widget#9]
Thanks a lot @gereleth for this helpful package. One thing that tripped me up: I tried using `on_submit` and `on_skip` as decorators. For example:

```python
widget = BBoxWidget(classes=[...])

@widget.on_skip
def advance():
   ...

@widget.on_submit
def submit():
   save()
   advance()
```

However, this did not work as intended, as `on_submit` and `on_skip` do return `None`. Maybe it would be an option to return the functions themselves?

**Comment by gereleth** [comment]
Hi, this seems like a good idea at first glance. I'll take a closer look in a few days.

**Comment by gereleth** [comment]
Thanks again for the suggestion, I implemented this in version 0.5.0.

**Comment by chmp** [comment]
Really cool. Looking forward to use it :)
